hs_core/templatetags/hydroshare_tags.py

```diff
@@ -205,6 +205,8 @@ def relative_irods_path(fed_irods_file_name):
 
 @register.filter
 def resource_from_uuid(id):
```

Comment:
this filter is only used in 403.html.  Without the id check, the 403 would be turned to a 404 as get_resource_by_shortkey raises a 404 if no id is supplied.